src/main/java/teammates/common/util/Templates.java

```diff
@@ -10,17 +10,18 @@ private Templates() {
     
     /**
      * Populates the HTML templates by replacing variables in the template string
-     * with the given value string.
+     * with the given value strings.
      * @param template The template html to be populated
-     * @param values Array of a variable, even number of key-value pairs:
+     * @param keyValuePairs Array of a variable, even number of key-value pairs:
      *                   { "key1", "val1", "key2", "val2", ... }
      * @return The populated template
      */
-    public static String populateTemplate(String template, String... values) {
-        Assumption.assertTrue("The number of values passed in must be even", values.length % 2 == 0);
+    public static String populateTemplate(String template, String... keyValuePairs) {
```

Comment:
Hmm i think you are right but such a change would involve too many other files. I will open a new issue for refactoring it.